### DISCUSSION TITLE
ValidationError: 1 validation error for BaseEvent

### DISCUSSION THREAD

**Discussion by BiffoQ** [deepanshs/mrsimulator#359] [Q&A] [answered]
Hi Deepansh,

I got to know about mrsimulator from Grandinetti's lecture at the just concluded PANACEA NMR Summer School in Venice.

So I am trying the package for the first time and I am running into this error after just trying to import signal_processor from mrsimulator

`from mrsimulator import signal_processor as sp`

I am getting this error:

ValidationError: 1 validation error for BaseEvent
rotor_frequency
  Input should be a valid number [type=float_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.7/v/float_type

**Comment by deepanshs** [comment]
Hi,

This is a know issue and will be fixed in the next release. In the meanwhile, please downgrade pydantic as follows

```shell
pip install "pydantic<2"
```

Let me know if you see other issues. Happy to help.

Best,
Deepansh

**Comment by BiffoQ** [comment, marked as answer]
Thanks for your response. It is working.
Cheers,
Abdulkadir.

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________
